practice/00-HTML-CSS-basics/form.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html> <!-- If this tag is missing, I just see that the upper margin is reduced. Must see the efect in the other documents -->
+<html lang="en">
+	<head>
+		<title>Form Page</title>
+		<meta charset="UTF-8">
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	</head>
+
+	<body>
+		<h1>Personal data</h1>
+		<!-- Validation rules are accomplished via HTML5's type property (email, date, etc.)-->
+		<form method="get" action="#">
+			<p>
+				<label for="first-name">First name(s):</label>
+				<input type="text" name="first-name">
+			</p>
```

Comment:
Don't surround your labels/inputs with the <p>, this is meant for text, if you want to wrap things separately you should use <div>